src/utils/path.ts

```diff
@@ -48,8 +48,27 @@ export const getNetworkConfigFilesPath = (
 	};
 };
 
-export const getConfigFilePath = (dataPath: string, network: string): string =>
-	path.join(dataPath, 'config', network, 'config.json');
+export const getDefaultNetworkConfigFilesPath = (
+	network: string,
+): { genesisBlockFilePath: string; configFilePath: string } => {
+	const basePath = path.join(getDefaultConfigDir(), 'config', network);
+	return {
+		genesisBlockFilePath: path.join(basePath, 'genesis_block.json'),
+		configFilePath: path.join(basePath, 'config.json'),
+	};
+};
+
+export const getConfigDir = (dataPath: string): string[] => {
```

Comment:
Its returning list of directories in the folder, so its name should be plural. or `geConfigDirs` or `getSubConfigDirs`

```suggestion
export const getNestedConfigDirs = (dataPath: string): string[] => {
```